### DISCUSSION TITLE
How does Berry compare to Lua ?

### DISCUSSION THREAD

**Discussion by AlixANNERAUD** [berry-lang/berry#313] [Q&A]
Hello,
Has anyone done benchmarks to know if the footprint of the berry (computation time, memory footprint ...) was better / worse than Lua? And if so, by how much?

**Comment by s-hadinger** [comment]
I don't think anybody did benchmarks, and it's not trivial to do such comparison. Berry is supposed to be simpler and more lightweight than LUA. I don't know about speed and memory usage.

**Comment by skiars** [comment]
I benchmarked it a long time ago, and Berry is slightly slower than Lua, but better than MicroPython, etc. Berry consumes far less memory (and less code) than Lua, which makes it easier to use on MCUs.
